public/stylesheets/style.css

```diff
@@ -123,6 +123,36 @@ div[class^="col-"] {
 .reviewAuthor {margin: 0 5px;}
 .reviewTime {color: #ccc;}
 
+/*CSS for custom footer ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/
+
+ul {
+  list-style-type: none;
+}
+
+a.bottomlink {
+    font-family: 'Roboto', sans-serif;
+    font-size:14px;
+    margin-right:110px;
+
+    padding:5px;
+    width:100px;
+    text-align:center;
+
+    color:black;
```

Comment:
I think a lighter color would be more appropriate for a footer but that's subjective.